external/mongodb/mongo.go

```diff
@@ -0,0 +1,22 @@
+package mongodb
+
+import (
+	"context"
+
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+	"go.mongodb.org/mongo-driver/mongo/readpref"
+)
+
+func ConnectMongoDB(clientOptions *options.ClientOptions) (*mongo.Client, error) {
```

Comment:
ConnectMongoDB -> NewConnection 
Is this better?

Reply:
I totally agree with you.